packages/snaps-simulator/src/features/simulation/snap-permissions.ts

```diff
@@ -0,0 +1,126 @@
+import { GenericPermissionController } from '@metamask/permission-controller';
+import {
+  restrictedMethodPermissionBuilders,
+  selectHooks,
+} from '@metamask/rpc-methods';
+import { endowmentPermissionBuilders } from '@metamask/snaps-controllers';
+import { DEFAULT_ENDOWMENTS } from '@metamask/snaps-utils';
+
+export const ExcludedSnapEndowments = Object.freeze(['endowment:keyring']);
+
+export const buildSnapEndowmentSpecifications = () =>
```

Comment:
Should be exported from `snaps-controllers` - but we can do that in a separate PR. Create an issue perhaps?